apps/foundation-demo/src/app/cart/components/cart-wrapper/cart-wrapper.component.html

```diff
@@ -2,15 +2,15 @@
   <div class="cart-wrapper__main">
     <div class="cart-wrapper__header">
       <span class="cart-wrapper__title">Your Cart</span>
-      <span class="cart-wrapper__item-count">{{cart.items.length}} {{itemText}}</span>
+      <span class="cart-wrapper__item-count">{{(itemCount$ | async)}} {{(itemText$ | async)}}</span>
```

Comment:
Is it possible to "pluralize" a word? I feel that's better than itemText being a part of a redux selector.

Reply:
Seems like just the right kind of scenario. I'll add an issue for it.